### PR TITLE
React redux API

### PR DESCRIPTION
**Pull request by fed1k**
* Save everything in the API
* Kepp the same functionality

### CODE REVIEW

File: src/components/Input.js [hunk 1]
```diff
@@ -28,6 +36,9 @@ const Input = () => {
       addBook();
     }
   };
+  // const store = useSelector((state) => state);
+
+  // dispatch(thunkFunction());
```
Comment (brytebee) [inline src/components/Input.js]: - [x] **Optional:** Kindly remove commented code from your codebase. It is considered bad practice to commit commented code. And it also makes code hard to read.

File: src/components/Books.js [hunk 2]
```diff
@@ -1,18 +1,32 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getDataFromAPi } from '../redux/books/books';
 import Book from './Book';
 import Input from './Input';
+// import { getDataFromAPi } from './..redux/books/books';
```
Comment (brytebee) [inline src/components/Books.js]: - [x] **Optional:** Kindly remove commented code from your codebase. It is considered bad practice to commit commented code. And it also makes code hard to read.

**Review by brytebee — APPROVED** [review]
Hi, @fed1k👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Books.js (hunk 2 above)
Comment (fed1k) [reply src/components/Books.js]: Hello @brytebee. Thanks for the code review. I will implement it and then merge it. Thanks again

**Pull request merged**